components/smaily/actions/update-subscriber/update-subscriber.ts

```diff
@@ -0,0 +1,43 @@
+import { defineAction } from "@pipedream/types";
+import smaily from "../../app/smaily.app";
+
+export default defineAction({
+  name: "Update Subscriber",
+  version: "0.0.1",
+  key: "smaily-update-subscriber",
+  description: "Updates a subscriber. [See docs here](https://smaily.com/help/api/subscribers-2/create-and-update-subscribers/)",
+  type: "action",
+  props: {
+    smaily,
+    email: {
+      label: "Subscriber Email",
+      description: "The email of the subscriber",
+      type: "string",
+    },
```

Comment:
Could the `email` prop use async options?
List subscribers - https://smaily.com/help/api/segments/list-subscribers-of-a-segment/
List segments - https://smaily.com/help/api/segments/list-segments/